### PR TITLE
App chart, RBAC/PSP for docs

### PR DESCRIPTION
**Pull request by marians**
Towards https://github.com/giantswarm/giantswarm/issues/7349

This migrates the former helm chart to the new app-operator compatible chart. It also adds RBAC/PSP things that were missing so far to run the workload without root privileges and use EmptyDir type
 volumes.

### CODE REVIEW

File: helm/docs-app/templates/psp-rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    rule: RunAsAny
+    supplementalGroups:
+    ranges:
+    - max: 65535
+        min: 1
+    rule: MustRunAs
+    volumes:
+    - secret
+    - configMap
+    - emptyDir
```
Comment (calvix) [inline helm/docs-app/templates/psp-rbac.yaml]: the indentation on psp s broken , check this for comparsion https://github.com/giantswarm/happa/blob/master/helm/happa-chart/templates/psp.yaml

Comment (marians) [reply helm/docs-app/templates/psp-rbac.yaml]: Thanks! Needed to change my editor's indentation settings from the default 4 spaces...

**Comment by marians** [comment]
I don't understand why the latest commits don't trigger any checks/builds in CircleCI. Glad for help.

**Comment by calvix** [comment]
> I don't understand why the latest commits don't trigger any checks/builds in CircleCI. Glad for help. 

you are missing few lines in order to use orbs you need version 2.1 and definition of orbs

https://github.com/giantswarm/k8s-api-healthz/blob/master/.circleci/config.yml#L1-L3

**Comment by marians** [comment]
Thank you!

File: helm/docs-app/templates/psp-rbac.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: extensions/v1beta1
```
Comment (oponder) [inline helm/docs-app/templates/psp-rbac.yaml]:
```suggestion
apiVersion: policy/v1beta1
```

**Comment by marians** [comment]
This is now running in a cluster for test purposes. See https://github.com/giantswarm/tenant-cluster-installations/pull/40 for the app manifest.

File: helm/docs-app/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        - name: {{ .Values.image.name }}
+          # Public image required
+          image: quay.io/giantswarm/{{ .Values.image.name }}:{{ .Values.image.sha }}
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8080
+              name: http
+          resources:
+            requests:
+              cpu: 0.2
```
Comment (calvix) [inline helm/docs-app/templates/deployment.yaml]: little inconsistency between  components, somewhere you use 200m and here 0.2

Comment (marians) [reply helm/docs-app/templates/deployment.yaml]: Done

**Pull request merged**